core/class/cmd.class.php

```diff
@@ -90,10 +149,12 @@ private static function cast($_inputs, $_eqLogic = null) {
 		return $_inputs;
 	}
 
-	/**
-	 * @param int|string $_id the id of the command
-	 * @return void|cmd void if $_id is not valid else the cmd
-	 */
+    /**
+     * Get a command by ID
+     *
+     * @param int|string $_id Command ID
+     * @return void|static Null if ID not valid, command object otherwise
```

Comment:
issue with return type, should be cmd and not static
same issue with others comments

Reply:
The use of `static` instead of `cmd` is intentional and more appropriate here. `static` refers to the current class or any class that inherits from it, which is particularly relevant in Jeedom where plugins can extend the `cmd` class. This allows for better flexibility and more accurate typing when a plugin class inherits from `cmd`.

Reply:
now I do remember having seen this "return static" syntax but I didn't realize the impact and advantage
that's really great, i like it
thanks for your answer